gems/rails/CVE-2009-3009.yml

```diff
@@ -0,0 +1,32 @@
+---
+gem: rails
+cve: 2009-3009
+osvdb: 57666
+ghsa: 8qrh-h9m2-5fvf
+url: http://weblog.rubyonrails.org/2009/9/4/xss-vulnerability-in-ruby-on-rails
```

Comment:
Same here. The patches reference `activesupport` and `actionpack`. Probably OK to copy this file into `gems/activesupport` and `gems/actionpack`.

Reply:
Made the changes.